tests/testPhotoCal.py

```diff
@@ -125,6 +124,29 @@ def _runTask(self):
         self.zp = pCal.calib.getMagnitude(1.)
         self.fitdiff = pCal.arrays.srcMag + self.zp - pCal.arrays.refMag
 
+    def testFlags(self):
+        """test that all the calib_photometry flags are set to reasonable values"""
+        schema = self.srcCat.schema
+        task = PhotoCalTask(self.refObjLoader, config=self.config, schema=schema)
+        mapper = afwTable.SchemaMapper(self.srcCat.schema, schema)
+        cat = afwTable.SourceCatalog(schema)
+        for name in self.srcCat.schema.getNames():
+            mapper.addMapping(self.srcCat.schema.find(name).key)
+        cat.extend(self.srcCat, mapper=mapper)
+
+        #   test that by default, no stars are reserved and used < candidates
+        task.run(exposure=self.exposure, sourceCat=cat)
+        candidates = cat.get("calib_photometryCandidate").sum()
+        assert(cat.get("calib_photometryReserved").sum() == 0)
+        assert(cat.get("calib_photometryUsed").sum() <= candidates)
```

Comment:
Though actually, I'd think you'd be able to know exactly how many objects are selected for `used` here: can't you test with that number, so we can know if it ever changes?